### PR TITLE
lab1 review

### PR DESCRIPTION
**Pull request by scollier**

### CODE REVIEW

File: labs/lab1/chapter1.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+What is [podman](https://github.com/projectatomic/libpod), you may ask? Well, the README explains it in detail but, in short, it is a tool for manipulating OCI compliant containers created by docker or other tools (such as buildah). The docker utility provides build, run, and push functions on docker containers via a docker daemon. We are leveraging three daemonless tools, which support OCI compliant containers, that do each function separately. Namely, [buildah](https://github.com/projectatomic/buildah) for building, podman for download, verification, and run, and [skopeo](https://github.com/projectatomic/skopeo) for distribution (pushing). You don't need the overhead of running a daemon to leverage these capabilities.
```
Comment (dustymabe) [inline labs/lab1/chapter1.md]: we could probably beat the "daemonless" drum a little less hard right here. I'm not trying to come off sounding too biased.

Comment (dustymabe) [reply labs/lab1/chapter1.md]: also, should we even mention skopeo? podman transparently uses the tech in the background right? do we ever call skopeo directly in the lab?

**Comment by dustymabe** [comment]
LGTM

File: labs/lab1/chapter1.md (hunk 1 above)
Comment (whitel) [reply labs/lab1/chapter1.md]: I didn't realize podman wrapped skopeo too. I was trying to point out the unix-philosophy of it.

**Pull request merged**